### PR TITLE
Added dependencies for MySQL client connections and updated config_up…

### PR DESCRIPTION
**Pull request by pidydx**
…dater warning

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -40,3 +40,4 @@ selenium==2.44.0
 urllib3==1.10
 wsgiref==0.1.2
 Werkzeug==0.11.3
+MySQL-python==1.2.5
```
Comment (destijl) [inline requirements.txt]: This library is small, 110k. It needs the mysql libs to install, so it won't work on os x or windows, but we have separate client building requirements.txt for those. So I think this is OK to go in.

File: scripts/install_script_ubuntu.sh [hunk 2]
```diff
@@ -149,6 +149,7 @@ sudo apt-get install -y \
   libfreetype6-dev \
   libpng-dev \
   libprotobuf-dev \
+  libmysqlclient-dev \
```
Comment (destijl) [inline scripts/install_script_ubuntu.sh]: This also needs to go into https://github.com/google/grr/blob/master/travis/before_install.sh

File: tools/config_updater.py [hunk 3]
```diff
@@ -501,6 +501,18 @@ def ConfigureDatastore(config):
...
+    print """\n\n***WARNING***
+
+Do not continue until a MySQL server, version 5.6 or greater, is running and a user
+with the ability to create the GRR database and tables has been created. You will need
+the server, username, password, and database name (if already created) to continue. If
+no database has been created this script will attempt to create the necessary database 
+and tables using the credentials provided.
+
+***WARNING***
+"""
```
Comment (destijl) [inline tools/config_updater.py]: Fix line breaks here.

Comment (pidydx) [inline tools/config_updater.py]: You mean use \n instead of literals?

Comment (destijl) [inline tools/config_updater.py]: Looks like it isn't wrapped at 80 correctly, there's breaks in the middle of sentences

Comment (pidydx) [inline tools/config_updater.py]: Does this look better?

File: scripts/install_script_ubuntu.sh (hunk 2 above)
Comment (pidydx) [inline scripts/install_script_ubuntu.sh]: Fixed

**Comment by destijl** [comment]
LGTM this can go in on next sync, assuming travis builds OK.

**Pull request merged**